### PR TITLE
io: first 3 tasks from input/output

### PR DESCRIPTION
**Pull request by aleksandrhristov**
The first 3 tasks of input and output: 
 * reversing file content 
 * reading from console and writing content to file till special character is entered 
 * helper class for reading from console

### CODE REVIEW

File: input-output/src/addtextinfile/TextFiles.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import java.io.*;
+import java.util.Scanner;
+
+/**
+ * Created by clouway on 10.05.16.
+ */
+public class TextFiles {
+
+    public void createFile(String name, String... names) {
+        console(name, names);
```
Comment (mgenov) [inline input-output/src/addtextinfile/TextFiles.java]: What is doing the console? prints messages or reads messages ?

File: input-output/src/reversecontent/TextFile.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package reversecontent;
+
+import java.io.*;
+
+/**
+ * Created by clouway on 10.05.16.
+ */
+public class TextFile {
+
+    public int[] Reverse (java.io.File Reverse) throws IOException {
```
Comment (mgenov) [inline input-output/src/reversecontent/TextFile.java]: Name of the methods should start with lower case character.

File: input-output/src/reversecontent/TextFile.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        int byteContainer[] = new int[currentByte];
+        while(readCount < currentByte){
+            byteContainer[readCount] = source.read();
+            readCount++;
+        }
+        source.close();
+        return byteContainer;
+    }
+
+    public void printReverse(int[] fileContent) throws IOException {
```
Comment (mgenov) [inline input-output/src/reversecontent/TextFile.java]: Mostly file content is passed as byte[] array and whole content could be written as single operation and much less boilerplate. 

https://docs.oracle.com/javase/7/docs/api/java/io/OutputStream.html#write(byte[])

File: input-output/src/reversecontent/Demo1.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+/**
+ * Created by clouway on 10.05.16.
+ */
+public class Demo1 {
+    public static void main(String[] args) throws IOException {
+        java.io.File fileToRead = new java.io.File("files.txt");
+
+        TextFile demo = new TextFile();
+        int[] readBytes = demo.Reverse(fileToRead);
```
Comment (mgenov) [inline input-output/src/reversecontent/Demo1.java]: Method names in java should start with lower case characters.

Comment (mgenov) [inline input-output/src/reversecontent/Demo1.java]: It's unreasonable textFile to reverse another file. 

Someting like: 
```java
TextFile usersDb = new TextFile("users.txt");
byte[] content = usersDb.reversedContent(); // returns the reversed content of file. 
```

is much more reasonable probably.

File: input-output/src/addtextinfile/TextFiles.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        for (int i = 0; i < stopSymbols.length; i++) {
+            if (symbol.equals(stopSymbols[i])) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+}
+
```
Comment (mgenov) [inline input-output/src/addtextinfile/TextFiles.java]: These empty lines need to be removed.

File: input-output/src/console/Console.java [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Created by clouway on 09.05.16.
+ */
+public class Console {
+    public String readString() throws IOException {
```
Comment (mgenov) [inline input-output/src/console/Console.java]: Doesn't looks good new instance to be created each time when line is going to be read.

File: input-output/src/addtextinfile/TextFiles.java (hunk 5 above)
Comment (aleksandrhristov) [inline input-output/src/addtextinfile/TextFiles.java]: fixed

**Pull request closed without merging**